internal/cli/options.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import "chainguard.dev/apko/pkg/log"
+
+type publishOpt struct {
+	packageVersionTag       string
+	packageVersionTagStem   bool
+	packageVersionTagPrefix string
+	tagSuffix               string
+	local                   bool
+	stageTags               string
+	tags                    []string
+	logger                  log.Logger
+}
+
+// PublishOption is an option for publishing
+type PublishOption func(*publishOpt) error
+
+func WithPackageVersionTag(pvt string) PublishOption {
+	return func(p *publishOpt) error {
+		p.packageVersionTag = pvt
+		return nil
+	}
+}
+
+func WithPackageVersionTagStem(packageVersionTagStem bool) PublishOption {
+	return func(p *publishOpt) error {
+		p.packageVersionTagStem = packageVersionTagStem
+		return nil
+	}
+}
+
+func WithPackageVersionTagPrefix(packageVersionTagPrefix string) PublishOption {
+	return func(p *publishOpt) error {
+		p.packageVersionTagPrefix = packageVersionTagPrefix
+		return nil
+	}
+}
+
+// withTagSuffix sets a tag suffix to use, e.g. `-glibc`.
```

Comment:
A bunch of these have lowercase comments but uppercase names.

Reply:
Fixed